templates/types/streaming/nextjs/app/components/ui/chat/chat-input.tsx

```diff
@@ -34,6 +36,13 @@ export default function ChatInput(
     reset,
     getAnnotations,
   } = useFile();
+  const { isUsingLLamaCloud, projects, pipeline, setPipeline } =
+    useLlamaCloud();
+
+  // Additional data to be sent to the API endpoint.
+  const requestAdditionalData = {
+    llamaCloudPipeline: pipeline,
+  };
```

Comment:
the changees to chat-input should be as minimal as possible (we just want to add the selector with a build flag):
```suggestion
  const [requestData, setRequestData] = useState(null);
```